onward/app/views/fragments/rightMostPopular.scala.html

```diff
@@ -9,7 +11,7 @@ <h3 class="content__meta-heading">Most popular</h3>
                     <a class="right-most-popular-item__url media u-cf" href="@LinkTo{@trail.url}">
                         @trail.trailPicture(5, 3).map{ image =>
                             <div class="right-most-popular-item__img media__img">
-                                <img class="responsiveimg" src="@Item120.bestFor(image)" alt="" />
+                                <img class="responsiveimg" src="@if(request.isInImgixTest && ImgixSwitch.isSwitchedOn) { @ImgSrc.getFallbackUrl(image) } else { @Item120.bestFor(image) }" alt="" />
```

Comment:
`ImgixSwitch.isSwitchedOn` is implicit in `request.isInImgixTest` (but I don't really care)